routes/UserSignIn.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+var path = require('path');
+
+/* GET home page. */
+router.get('/', function(req, res) {
```

Comment:
put this route inside index.js
it would look something like `router.get('/badgein', function(req, res) {});`